programs/uxd/tests/integration_tests/api/program_spl/accounts/read_token_account.rs

```diff
@@ -0,0 +1,19 @@
+use solana_program::program_pack::Pack;
+use solana_program::pubkey::Pubkey;
+use solana_program_test::ProgramTestContext;
+
+pub async fn read_token_account(
+    program_test_context: &mut ProgramTestContext,
+    token_account: &Pubkey,
+) -> Result<spl_token::state::Account, String> {
+    let raw_account = program_test_context
+        .banks_client
+        .get_account(*token_account)
+        .await
+        .map_err(|e| e.to_string())?
+        .ok_or("AccountDoesNotExist")?;
+    if raw_account.owner != spl_token::id() {
+        return Err(String::from("InvalidTokenAccount"));
```

Comment:
Could create `error.rs` solely for tests purposes

Reply:
👌 done in previous PR